packages/synapse-react-client/src/utils/hooks/useScrollFadeTransition.ts

```diff
@@ -6,7 +6,8 @@ const calculateOpacity = (rect: DOMRect): number => {
   const middleRangeStart = viewportHeight * 0.4
   const middleRangeEnd = viewportHeight * 0.6
 
-  if (elementCenterY >= middleRangeStart && elementCenterY <= middleRangeEnd) {
+  if (elementCenterY <= middleRangeEnd) {
+    // && elementCenterY >= middleRangeStart) {
```

Comment:
was this commented out on purpose?

Reply:
Yes.  Adam would like to slightly change the fade in behavior (keep it solid/opaque as you scroll down the page past the element)